quiz.js

```diff
@@ -49,6 +74,11 @@ db.dropDatabase();
 
 // your code begins here
 
+db.houses.update(
+  {house: "House Arryn"},
+  {$set: {motto: ""}}
```

Comment:
This will set the value of the motto to an empty string, but won't delete the motto property. To do that, you'd need to use [$unset](http://docs.mongodb.org/manual/reference/operator/update/unset/):
`db.houses.update( { house: 'House Arryn' },  { $unset: { motto: "" } } );`

However, that may not have been clear from the question, so no points off for this.